### DISCUSSION TITLE
New deploy 02/05 has a bug

### DISCUSSION THREAD

**Discussion by isolution1** [oqtane/oqtane.framework#1094] [Q&A]
When running new deploy we receive the following error InvalidOperationException: Unable to resolve service for type 'Oqtane.Infrastructure.LocalizationManager' while attempting to activate 'Oqtane.Pages.HostModel'.
I could not fix this

**Comment by hishamco** [comment]
Thanks @isolution1 for reporting this, is this happening with a new installation?

**Reply by hishamco** [reply]
Seems the `ILocalizationManager` in #1071 not registered at that point of time, I will try to check few cases

Thanks

**Reply by hishamco** [reply]
I just found the issue it should be `ILocalizationManager` not `LocalizationManager`, PR is on the way ...

**Comment by hishamco** [comment]
Should be fixed in #1095

**Comment by sbwalker** [comment]
@isolution1 @hishamco this is now fully fixed in #1096

**Reply by hishamco** [reply]
Thanks Shaun, this should be resolved then
